src/core/tests/Test_AptitudePackageManager.py

```diff
@@ -349,6 +373,117 @@ def test_update_image_default_patch_mode_raises_exception(self):
         self.runtime.env_layer.file_system.write_with_retry = self.mock_write_with_retry_raise_exception
         self.assertRaises(Exception, package_manager.update_os_patch_configuration_sub_setting)
 
+    def test_is_reboot_pending_prerequisite_not_met_should_return_false(self):
+        package_manager = self.container.get('package_manager')
+        package_manager._AptitudePackageManager__pro_client_prereq_met = False
+
+        self.assertFalse(package_manager.is_reboot_pending())
+
+    def test_is_reboot_pending_prerequisite_met_should_return_true(self):
+        package_manager = self.container.get('package_manager')
+        package_manager._AptitudePackageManager__pro_client_prereq_met = True
+
+        self.assertTrue(True, package_manager.is_reboot_pending())
```

Comment:
Here, the statement should be self.assertTrue(package_manager.is_reboot_pending()) or self.assertEqual(True, package_manager.is_reboot_pending()).

I did debug and found that here package_manager.is_reboot_pending() is returning False because of exception while finding reboot status using pro.

Reply:
done. Thanks